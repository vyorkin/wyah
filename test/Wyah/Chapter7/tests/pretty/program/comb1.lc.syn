Program [Decl "W" (ELam (Var "x") (ELam (Var "y") (EApp (EApp (EVar (Var "x")) (EVar (Var "y"))) (EVar (Var "y"))))),Decl "C" (ELam (Var "x") (ELam (Var "y") (ELam (Var "z") (EApp (EApp (EVar (Var "x")) (EVar (Var "z"))) (EVar (Var "y")))))),Decl "B" (ELam (Var "x") (ELam (Var "y") (ELam (Var "z") (EApp (EVar (Var "x")) (EApp (EVar (Var "y")) (EVar (Var "z")))))))]
