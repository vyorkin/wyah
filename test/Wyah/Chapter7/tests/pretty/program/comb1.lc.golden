let W = \x -> \y -> (x y) y;
let C = \x -> \y -> \z -> (x z) y;
let B = \x -> \y -> \z -> x (y z);